modules/appeals_api/spec/factories/higher_level_reviews.rb

```diff
@@ -52,14 +52,14 @@
   factory :extra_higher_level_review_v0,
           class: 'AppealsApi::HigherLevelReview', parent: :extra_higher_level_review_v2 do
     api_version { 'V0' }
-    auth_headers { FixtureHelpers.fixture_as_json 'higher_level_reviews/v0/valid_200996_headers_extra.json' }
```

Comment:
Previously, this extra headers fixture was used in this factory because it had `X-VA-ICN` (which was newly required in v0). Now that this has moved to the body it can use the default headers (which only contain the consumer username and ID)